java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1099,6 +1100,10 @@ class DatabaseImpl(
             """)
     }
 
+    override suspend fun getEntitiesCount(): Long {
+        return DatabaseUtils.queryNumEntries(readableDatabase, "entities")
```

Comment:
there is a const for the table name: TABLE_ENTITIES

Reply:
Done.